UI/WebServerResources/SOGoMailDataSource.js

```diff
@@ -82,9 +82,35 @@ SOGoMailDataSource = Class.create({
         },
 
         _loadCallback: function(http) {
+            var data,
+            re, reMatches, formLine;
             if (http.status == 200) {
                 if (http.responseText.length > 0) {
-                    var data = http.responseText.evalJSON(true);
+                    try {
+                        data = http.responseText.evalJSON(true);
+                    } catch(e) {
```

Comment:
The correct design would be that instead of returning the login page to return the error code + the url to redirect. So it would not be needed to check the content header.
More likely this design would not be materialize but I would like to keep the "exceptional" nature of not finding json data.